OracleWebCenterPortal/dockerfiles/README.md

```diff
@@ -0,0 +1,123 @@
+
+This project creates Oracle WebCenter Portal Docker image with a single node targeted for development and testing.
+
+# 2. Hardware and Software Requirements
+Oracle WebCenter Portal has been tested and is known to run on the following hardware and software:
+
+## 2.1 Hardware Requirements
+
+| Hardware  | Size  |
+| :-------: | :---: |
+| RAM       | 16GB  |
+| Disk Space| 200GB+|
+
+## 2.2 Software Requirements
+
+|       | Version                        | Command to verify version |
+| :---: | :----------------------------: | :-----------------------: |
+| OS    | Oracle Linux 7.3 or higher     | more /etc/oracle-release  |
+| Docker| Docker version 17.03 or higher | docker version           |
+
+# 3. Prerequisites
+## 3.1. Mount a host directory as a data volume
+
+You need to mount volumes, which are directories stored outside a container’s file system, to store WebLogic domain files and any other configuration files.
+
+To mount a host directory `/scratch/wcpdocker/volumes/wcpportal`($DATA_VOLUME) as a data volume, execute the below command.
+
+> The userid can be anything but it must belong to uid:guid as 1000:1000, which is same as 'oracle' user running in the container.
+
+> This ensures 'oracle' user has access to shared volume.
+
+```
+$ sudo mkdir -p /scratch/wcpdocker/volumes/wcpportal
+$ sudo chown 1000:1000 /scratch/wcpdocker/volumes/wcpportal
+```
+
+All container operations are performed as **'oracle'** user.
+
+**Note**: If a user already exist with **'-u 1000 -g 1000'** then use the same user. Or modify any existing user to have uid-gid as **'-u 1000 -g 1000'**
+
+## 3.2. Database
+You need to have a running database container or a database running on any machine. 
+The database connection details are required for creating WebCenter Portal specific RCU schemas while configuring WebCenter Portal domain. 
+
+## 3.3 Oracle Fusion Middleware Infrastructure image
+>1. Sign in to Oracle Container Registry. Click the Sign in link which is on the top-right of the Web page.
+> 2. Click Middleware and then click Continue for the fmw-infrastructure repository.
+> 3. Click Accept to accept the license agreement.
+> 4. Use following commands to pull Oracle Fusion Middleware infrastructure base image from repository :
+```
+$ docker login container-registry.oracle.com
+$ docker pull container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4
+$ docker tag container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4 oracle/fmw-infrastructure:12.2.1.4.0
+```
+
+
+The Oracle Database image can be pulled from the [Docker Store](https://store.docker.com/images/oracle-database-enterprise-edition) or the [Oracle Container Registry](https://container-registry.oracle.com) or you can build your own using the Dockerfiles and scripts in [GitHub Location](https://github.com/oracle/docker-images/tree/master/OracleDatabase).
```

Comment:
Please remove all references to the Docker Store images as they are no longer maintained. You should only ever reference the images on Oracle Container Registry.